site/content/help/reference/asking.md

```diff
@@ -7,7 +7,7 @@ description: "How to properly ask in our forums or support channel for assistanc
 * **Read the [Quickstart Guide](https://yuzu-emu.org/help/quickstart/).** Watching a YouTube video tutorial or reading a guide elsewhere doesn't count, as the majority of those provides incorrect instructions and quickly get outdated.
 * **Check if the problem is already reported in our [game compatibility list](https://yuzu-emu.org/game/)** Someone may be experiencing the same issue as you, or specific settings may be required for the game you are attempting to play.
 * **Reset your config file to defaults** This can resolve many common misconfiguration issues. This can be done by going to Configuration menu, General tab, and selecting "Reset All Settings"
-* **Make sure you're using the latest yuzu Canary build.** We don't provide support for older builds. Make sure you're using the latest Canary build available, as we always pour new improvements into them.
+* **Make sure you're using the latest yuzu mainline build.** We don't provide support for older builds. Make sure you're using the latest Canary build available, as we always pour new improvements into them.
```

Comment:
`mainline` should be `Mainline`

Also, at the end, it still says `you're using the latest Canary build available...`